main/main.cpp

```diff
@@ -2841,8 +2841,10 @@ Error Main::setup2() {
 
 	OS::get_singleton()->benchmark_end_measure("Startup", "Servers");
 
+#ifndef WEB_ENABLED
 	// Add a blank line for readability.
 	Engine::get_singleton()->print_header("");
```

Comment:
Is `Engine::get_singleton()->print_header(" ");` (space instead of empty) working in web export, or it will do the same? It's probably better to actually have a blank line, and should work on all platforms.

Reply:
The inserted space removes the appearance of the ‘empty string’ message.

This is the simplest solution (1-character commit :roll_eyes:), on the other hand, in the case of the browser console, usually each separate line of the log means something, hence my search for the source of the message (I thought the engine was trying to communicate something but the content was missing).